modules/fluent-bit/values.yaml

```diff
@@ -15,6 +15,26 @@ config:
         Systemd_Filter _SYSTEMD_UNIT=kubelet.service
         Read_From_Tail On
 
+  ## https://docs.fluentbit.io/manual/pipeline/filters
+  filters: |
+    [FILTER]
+        Name kubernetes
+        Match kube.*
+        Merge_Log On
+        Keep_Log Off
+        K8S-Logging.Parser On
+        K8S-Logging.Exclude On
+
+    [FILTER]
+        Name          grep
+        Match         *
+        Exclude       $log (ELB-HealthChecker|Amazon-Route53-Health-Check-Service|Health)
```

Comment:
Can we also have additional values supplied here to filter out some extra logs?
Also we should here filter prometheus and health with smaller H as seems this is case sensitive.

Reply:
Please have this as a list.